workflows/proteomics/clinicalmp/clinicalmp-data-interpretation/WF5_Data_Interpretation_Worklow_tests.yml

```diff
@@ -0,0 +1,67 @@
+- doc: Test outline for WF5_Data_Interpretation_Worklow
+  job:
+    Quantified_Peptides:
```

Comment:
Can you adjust all of the input and output labels to be more "human friendly" ? You can use a normal whitespace here, no need for - or _, and please also no all uppercase labels. These labels are shown in the invocation component and the workflow report, so it's nice if they look good.
You would need to change them here and in the workflow .ga file.

Generally speaking you also don't need to include the filetype extension in the label, the right filetype is shown in the user interface.